.github/workflows/check.yml

```diff
@@ -0,0 +1,17 @@
+name: check
+on:
+  pull_request:
+  push:
+jobs:
+  pre-commit:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v1
+      - run: pip install tox pre-commit
+      - run: pip install -r requirements.txt -r test-requirements.txt
```

Comment:
It is usually safer to edit `test-requirements.txt` to have the first line of be `-r requirements.txt` but this will work as well.